docs/build-tools/maven.md

```diff
@@ -0,0 +1,80 @@
+---
+id: maven
+title: maven
+---
+
+Maven is one of the most common build tools in the JVM ecosystem and it also allows for using scala through the [scala-maven-plugin](https://davidb.github.io/scala-maven-plugin/usage.html)
+
+## Automatic installation
+
+The first time you open Metals in a new workspace it prompts you to import the
+build. Select "Import build" to start automatic installation.
+
+This will create all the needed bloop files, however there will be a warning, since the semanticDB plugin was not added. Most features should work without it, however some require semanticDB files to be provided alongside compiled data. To do that we need a couple of steps that are explained in the manual instalation section.
+
+## Manual instalation
+
+First, add the following in the project's `pom.xml` `<dependencies>` section:
+
+```xml
+    <dependency>
+      <groupId>org.scalameta</groupId>
+      <artifactId>semanticdb-scalac_${scala.version}</artifactId>
+      <version>4.1.4</version>
+    </dependency>
+```
+`4.1.4` version can be replaced with whatever is the newest version.
+
+Now, run `mvn compile`, which will download the needed scalac plugin. The jar should be available in your repository, which usually is `<user_dir>/.m2`.
+
+Next, we need to add a couple of options to the compiler, that's why we will have to modify the configuration section of the plugin:
+
+```xml
+  <plugin>
+    <groupId>net.alchim31.maven</groupId>
+    <artifactId>scala-maven-plugin</artifactId>
+    <version>4.0.2</version>
+    <configuration>
+      <args>
+        <arg>-Xplugin:@@replace_with_semanticDB_path@@</arg>
```

Comment:
Using the absolute local path seems fragile, is there a Scala maven plugin that makes it easier to add a compiler plugin?

Reply:
scalor-maven-plugin: https://random-maven.github.io/scalor-maven-plugin/2.12/eclipse-config-mojo.html#definePluginList

scala-maven-plugin: http://davidb.github.io/scala-maven-plugin/example_scalac_plugins.html (see "using a scalac plugin"

Reply:
Och alright, I haven't noticed this solution. That makes it much easier now.